.circleci/config.yml

```diff
@@ -136,12 +139,22 @@ commands:
             - install-brew-dependency:
                 dependency_name: 'xcbeautify'
       - install-brew-dependency:
-          dependency_name: 'swiftlint'
+          dependency_name: 'mint'
+      - when:
+          condition:
+            not:
+              equal: ["", << parameters.swiftlint_version >>]
+          steps:
+            - run:
+                name: Install swiftlint@<< parameters.swiftlint_version >> using Mint 
+                command: |
+                  mint install realm/swiftlint@<< parameters.swiftlint_version >>
```

Comment:
Hmm I'm wondering, should we remove installing swiftlint from `install-dependencies` and just install it for the linter job? I think that we don't really need to install it for all jobs? That way we can just keep installing the latest version only, I don't see that much value in running swiftlint multiple times or anything.

Reply:
We _shouldn't_ need it for all jobs but it's part of a build phase or something IIRC, we'd need to remove it from there too. Otherwise a bunch of jobs will fail.

Reply:
Ohhh right... I guess we could skip that build phase if swiftlint is not installed, but yeah, I think this makes sense for now then 👍